demisto_sdk/commands/test_content/TestContentClasses.py

```diff
@@ -357,7 +362,7 @@ def create_incident(self, client: DefaultApi) -> Optional[Incident]:
             - The created incident or None
         """
         # Preparing the incident request
-        incident_name = f'inc-{self.configuration.playbook_id}'
+        incident_name = f'inc-{self.configuration.playbook_id}--{uuid.uuid4()}'
```

Comment:
is this required?

Reply:
Yes, since calling `create_incident` didnt return id of created incident, to get the id I calling to `find_incident` with query: name=incident_name. I want to be sure that I found the only one new created incident. So I added uuid.